youtube_dl/__init__.py

```diff
@@ -11,6 +11,10 @@
 import random
 import sys
 
+import gi
```

Comment:
This must be a soft dependency.

Reply:
How do you do this in this project - with `try:`/`except ImportError:`? I can't find anything like requirements files or `install_requires` in setup.py.

Reply:
Here's an example:
https://github.com/rg3/youtube-dl/blob/909191de9154bf289b333cfe01b8e88e3ac1fefc/youtube_dl/downloader/hls.py#L5-L9